test/Autofac.Integration.AspNetCore.Multitenant.Test/MultitenantRequestServicesMiddlewareTests.cs

```diff
@@ -17,31 +18,18 @@ public async Task Invoke_DoesNotOverrideExistingHttpContextOnAccessor()
         {
             var accessor = Mock.Of<IHttpContextAccessor>();
             accessor.HttpContext = new DefaultHttpContext();
-            var mtc = CreateContainer();
             var next = new RequestDelegate(ctx => Task.FromResult(0));
             var context = CreateContext();
 
-            var mw = new MultitenantRequestServicesMiddleware(next, accessor, new AutofacMultitenantServiceProvider(mtc));
+            var mw = new MultitenantRequestServicesMiddleware(next, accessor, CreateServiceProvider());
             await mw.Invoke(context);
             Assert.NotSame(context, accessor.HttpContext);
         }
 
-        [Fact]
```

Comment:
The middleware still needs to throw if there's no multitenant container. Did this test move elsewhere?

Reply:
Re-added on my local branch. Was a mistake, sorry.